File: flames.py
k1=raw_input('Enter your name')
print '\n'
k2=raw_input('Enter your partner name:')
s1=k1.lower()
s2=k2.lower()
str1=list(s1)
str2=list(s2)
for x in str1:
	for y in str2:
		if (x==y):
			del str1[str1.index(x)]
			del str2[str2.index(y)]
			break
a=len(str1)+len(str2)
if (a==3) or (a==14) or (a==16) or (a==18):
	print 'Autually you are Friends!!'
if (a==5) or (a==10):
	print 'WOW!! You are Lovers!!'
if (a==8) or (a==12) or (a==13) or (a==17):
	print 'Autually you have a Affection on her!!'
if (a==6) or (a==11) or (a==15):
	print 'You will Marry soon!!'
if (a==2) or (a==4) or (a==7) or (a==9) or (a==20):
	print 'OMG!! You are Enemies!!'
if (a==1) or (a==19):
	print 'Shitt!! She is Your sister!!'

File: guvi/C file/flames.c
#include<stdio.h>
#include<string.h>
void main()
{
	char a[200],b[200],f[10]="flames",f1[10];
	int l1,l2,l3,l4,i,t1,j,k,p,m,q,flag=0,n;
	printf("Enter your name:");
	gets(a);
	printf("\nEnter your partner name:");
    gets(b);
    l1=strlen(a);
    l2=strlen(b);
	for(i=0;i<l1;i++)
	{
		for(j=0;j<l2;j++)
		{
			if(a[i]==b[j])
			{
				a[i]='*';
				b[j]='*';
				break;
			}
		}
	}
	for(i=0;i<l1;i++)
	{
		if(a[i]!='*')
		{
			p++;
		}
	}
	for(i=0;i<l2;i++)
	{
		if(b[i]!='*')
		{
			q++;
		}
	}
	if(p+q==0)
	{
		printf("\nYou entered same name!!!!");
	}
	else if(p+q==1)
	{
		printf("\nYou are sister or brother!!!!");
	}
	else
	{
	for(k=0;k<5;k++)
	{
		i=0;
		t1=0;
		n=1;
		if(flag==1)
		{
		l3=strlen(f);
		for(m=0;m<l3;m++)
		{
			f[m]='\0';
		}
		l4=strlen(f1);
		for(m=0;m<l4;m++)
		{
			f[m]=f1[m];
		}
		for(m=0;m<l4;m++)
		{
			f1[m]='\0';
		}	
		}
		l3=strlen(f);
		while(n<=p+q)
		{
		flag=1;
		if(i==l3-1)
		{
			i=0;
			}	
		else{
			i++;
		}
		n++;
		if(n==p+q)
		{
			for(m=i+1;m<l3;m++)
			{
			f1[t1++]=f[m];	
			}
			for(m=0;m<i;m++)
			{
			f1[t1++]=f[m];	
			}
		}
		}
	}
	if(f1[0]=='f')
	{
		printf("\nYou are friends!!!!");
	}
	else if(f1[0]=='l')
	{
		printf("\nYou are Lovers!!!!");
	}
	else if(f1[0]=='a')
	{
		printf("\nYou have affection!!!!");
	}
	else if(f1[0]=='m')
	{
		printf("\nYou will marry!!!!");
	}
	else if(f1[0]=='e')
	{
		printf("\nYou are Enemies!!!!");
	}
	else if(f1[0]=='s')
	{
		printf("\nYou are sister!!!!");
	}

}
}
